src/content/articles/en/developing-meeting-culture-employees-love.mdx

```diff
@@ -0,0 +1,107 @@
+---
+id: 201911031
+title: "Developing a meeting culture that your employees will love"
+date: 2019-11-03T09:51:10+00:00
+author: Rathes Sachchithananthan
+image: ../../images/blog/meeting.jpg
+description: Do you like meetings? Do your employees? Creating a meeting culture that is efficient and that everybody loves isn't that difficult if you follow these ideas.
+categories:
+  - Business
+tags:
+  - Meetings
+  - Company Culture
+locale: en_US
+---
+
+import Note from "../../../components/Note"
+import Calculation from "../../../components/Calculation"
+
+# Developing a meeting culture that your employees will love
+
+Be honest to yourself. Do you like meetings in your business? Most of the people I talk to really hate meetings. They are unproductive, they are time-consuming, they are tiring. But if you dig deeper, the contents of those meetings are unavoidable. It is the meetings that lead to frustrations.
+Let me provide you a way how you can improve your meetings culture so that your employees, and colleagues won’t hate them anymore.
+
+<Note>
+  Note: The following ideas come from experiences working in and with
+  distributed teams but you can apply all of them to co-located teams as well.
+</Note>
+
+## 1. Do you need a meeting for this?
+
+Nobody likes meetings, but some meetings are crucial to success. It is just that not everything needs to be discussed in a meeting.
+Do you need to inform someone? Or multiple people? Why not send them are e-mail? Or a message in Slack? That is way faster and since it is async it will not interrupt the people you are informing.
```

Comment:
```suggestion
Do you need to inform someone? Or multiple people? Why not send them an e-mail? Or a message in Slack? That is way faster and since it is asynchronous it will not interrupt the people you are informing.
```